modules/nf-core/catpack/prepare/main.nf

```diff
@@ -14,16 +14,16 @@ process CATPACK_PREPARE {
     path acc2tax
 
     output:
-    tuple val(meta), path("${prefix}/db/")  , emit: db
-    tuple val(meta), path("${prefix}/tax/") , emit: taxonomy
-    path "versions.yml"                              , emit: versions
+    tuple val(meta), path("${prefix}/db/"), emit: db
+    tuple val(meta), path("${prefix}/tax/"), emit: taxonomy
+    path "versions.yml", emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
-    def args   = task.ext.args ?: ''
-    def prefix = task.ext.prefix ?: "${meta.id}"
+    def args = task.ext.args ?: ''
+    prefix = task.ext.prefix ?: "${meta.id}"
```

Comment:
why are these changes being reverted ?

Reply:
Because `$prefix`  is used in in the output definitons leading to test failure as they can't find them